### ISSUE TITLE
Original 21 games gone after update!?

### ISSUE THREAD

**Issue by Jenslyn87** [ClusterM/hakchi2#669]
Something happened with my original batch of games with the installation of the new version, all 21 of them. Now they don't take up ANY space at all. They show a black icon in the SNES UI and when I try to start them I get the C7 error. What could have gone wrong? :(. Emulated games show like normal and work just fine. Maybe I will have to flash the original kernel ... damn

Please, I'd appreciate any help you can offer.

**Comment by brian91292** [comment]
Just install original kernel then back to custom... should be fixed after that.

**Comment by Jenslyn87** [comment]
Thank you very much! I'll try this at once ... BRB

**Comment by Jenslyn87** [comment]
Well ... it partly worked. I flashed the original kernel, then flashed the custom one I want again and synchronized the games. I could tell something was off cause the synchronization went way too fast. At least I think so. I went to boot up the SNES and it was just reset to stock, factory 21 games functioning fine, but all my additional games/folders did not make it onto the system. I have no idea why. Any idea? Appreciate your help!

**Comment by Jenslyn87** [comment]
I notice that I can't get a green light on the SNES in Hackchi. Doesn't seem to connect. And when I select Sync now and it gives me the three-step prompt, nothing happens when I connect the SNES :(

**Comment by rsn8887** [comment]
Did you click "Synchronize Games" after you flashed the custom kernel again? For me that always fixes this problem.

**Comment by Jenslyn87** [comment]
Yes, I have tried that, but thanks a lot for trying to help!

There seems to be an issue, I think, where the SNES really never gets proper access to Hackchi. The dot in the bottom left only goes green ONLY right after I've flashed a new kernel. Otherwise it's always red. If I then sync the games, it goes very fast indeed, so fast that nothing really could be happening, and indeed nothing is. When I boot the system up, only the original games are there. If I sync the games without flashing the kernel immediately prior, I get the standard 3-point prompt asking me to turn on the SNES mini. If I do so, nothing happens. If I try to install the driver, I get a prompt saying, oddly, that no NES mini is detected. Any help I would really appreciate!

**Comment by Jenslyn87** [comment]
OK, I think I got this one solved with the hlep of you guys. Turns out after I synced the games (which for some reason I think I still had to do after flashing the custom kernel?) I quite simply had to wait. I suppose the SNES is internally finalizing the installation. I don't know. Right after the syncing was complete, the icon turned red. Then I just waited 3-45 seconds, and it turned green. Then I shut off the SNES and disconnected it. I then plugged it into my TV, and lo and behold, everything was in order. Strange, but this at least worked for me...

Thank you, all!
